hw/xilinx/ips/hpc/xlnx_ddr4/config.tcl

```diff
@@ -0,0 +1,20 @@
+# Author: Manuel Maddaluno <manuel.maddaluno@unina.it>
+# Description: DDR4 SDRAM (MIG) IP configuration file
+
+create_ip -name ddr4 -vendor xilinx.com -library ip -version 2.2 -module_name $::env(IP_NAME)
+
+set_property -dict [list \
+    CONFIG.System_Clock {Differential} \
+    CONFIG.C0.DDR4_AUTO_AP_COL_A3 {true} \
+    CONFIG.C0.DDR4_AxiAddressWidth {34} \
+    CONFIG.C0.DDR4_AxiDataWidth {512} \
+    CONFIG.C0.DDR4_AxiIDWidth {2} \
```

Comment:
for DDR4_AxiIDWidth, use config-based envvar, like in https://github.com/MaistoV/UninaSoC/blob/main/hw/xilinx/ips/embedded/xlnx_jtag_axi/config.tcl#L14

Reply:
see https://github.com/MaistoV/UninaSoC/pull/25#discussion_r1886852479